thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/validators/ConfigValidator.java

```diff
@@ -55,7 +55,7 @@ public boolean validateYAMLConfig(String yamlConfig) throws ValidationException
     try {
       Map<String, Object> yamlConfigMap = (Map<String, Object>) YAML.load(yamlConfig);
```

Comment:
Looks like this yamlConfig is loaded and thrown away immediately. It should be fine to let the client code to do validate yaml syntax, cause it has to load the yaml anyways.